test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -9,7 +9,7 @@ n_monitor_nodes: 1
 instance_type_db: 'i3.4xlarge'
 gce_instance_type_db: 'n1-highmem-16'
 gce_instance_type_loader: 'e2-standard-4'
-scylla_repo_loader: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-4.3.repo'
+scylla_repo_loader: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-4.6.repo'
```

Comment:
Why is it here and not in test defaults?

Reply:
on `test_default.yaml` it is empty, as for AWS we use a pre-installed loader, and it is needed only for tests that run on GCE (like 3h job)...
but yeah, most likely we should set it on the `test_default.yaml` only..

Reply:
@roydahan , fixed..